.github/workflows/partial-backend.yml

```
@@ -27,6 +27,12 @@ jobs:
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
         ports:
           - 5432:5432
+      ldap:
```

```suggestion
      ldap-service:
```

Just suggesting that the name match my suggestion on line 92.